app/setup/styleguide.styles.js

```diff
@@ -32,6 +34,28 @@ module.exports = {
         fontSize: 20
       }
     },
+    Heading: {
+      heading1: {
+        color: '#2c68a0',
+        width: '100%',
+        textAlign: 'center'
+      },
+      heading2: {
+        'backgroundColor': '#2c68a0',
+        'color': '#fff',
+        'width': 'calc(100% + 32px)',
+        'padding': '0px 15px',
+        'marginLeft': '-33px',
```

Comment:
- [ ] This is causing problems when the navigation disapears. Maybe you can extend the background by adding a pseudo element and remove the negative margin? There is a `rsg--hasSidebar-2` on a parent element if the sidebar is visible. Maybe this also helps.

Reply:
The problem doesn't occur anymore because of the changed subtitle alignment